pkg/storage/replica_range_lease.go

```diff
@@ -127,7 +129,11 @@ func (p *pendingLeaseRequest) InitOrJoinRequest(
 		// Get the liveness for the next lease holder and set the epoch in the lease request.
 		liveness, err := repl.store.cfg.NodeLiveness.GetLiveness(nextLeaseHolder.NodeID)
 		if err != nil {
-			llChan <- roachpb.NewErrorf("couldn't request lease for %+v: %v", nextLeaseHolder, err)
+			llChan <- roachpb.NewError(&roachpb.LeaseRejectedError{
```

Comment:
Because changing the default for epoch-range leases to "true" made the `TestNodeRestart` acceptance test fail regularly because it's slow enough that the node didn't have a chance to heartbeat the liveness table before the lease attempt failed. Because this error is generic, it bubbled up to the test caller and caused a failure. We might have been seeing this on our clusters, but it would just be some ignored errors at the block-writers and we probably wouldn't have seen them.